images/nginx/build.sh

```diff
@@ -283,14 +283,23 @@ cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF ..
 make
 make install
 
-# build zipkin lib
+# build jaeger lib
 cd "$BUILD_PATH/jaeger-client-cpp-$JAEGER_VERSION"
 sed -i 's/-Werror//' CMakeLists.txt
 mkdir .build
 cd .build
-cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=1 -DBUILD_TESTING=OFF -DJAEGERTRACING_WITH_YAML_CPP=OFF -DJAEGERTRACING_BUILD_EXAMPLES=OFF ..
+# Taken from https://github.com/jaegertracing/jaeger-client-cpp/blob/v0.4.1/scripts/build-plugin.sh
```

Comment:
@rnburn is this ok? I was using the same steps in your repository https://github.com/opentracing-contrib/nginx-opentracing/blob/master/Dockerfile#L65

Reply:
That looks good.

Jaeger needs the `JAEGERTRACING_WITH_YAML_CPP` option ena bled to be able to load a JSON configuration, which was probably what was causing trouble.

But BTW - since Jaeger links in everything statically, you should be able to remove the `_3rdParty/Hunter/` directory when you're finished. It would make your image smaller.

Reply:
@mikebryant can you implement what @rnburn mentioned?

Reply:
@mikebryant last change and we are ready to merge

Reply:
@aledbf Isn't that already taken care of by the `rm -rf "$BUILD_PATH"` line later in the script, that cleans up all of the build stuff?

Reply:
Maybe most of it's taken care of, but you can delete this line
```
cp $HUNTER_INSTALL_DIR/lib/libthrift* /usr/local/lib
```

Reply:
Ah, I see, sure

Reply:
Done :)